integration/src/main/java/scripts/utils/WorkspaceAllocateTestScriptBase.java

```diff
@@ -72,11 +74,18 @@ protected void doSetup(List<TestUserSpecification> testUsers, WorkspaceApi works
    */
   protected CreatedWorkspace createWorkspace(
       UUID workspaceUuid, String spendProfileId, WorkspaceApi workspaceApi) throws Exception {
+    Properties properties = new Properties();
```

Comment:
can you extract that as a public static field

Reply:
you also need to add the test in CloneWorkspace.java to assert that the properties is the same after cloning